src/KM_Points.pas

```diff
@@ -113,6 +113,7 @@   TKMTriMesh = record
 
 const
   ZERO_POINT: TKMPoint = (X: 0; Y: 0);
+  ZERO_POINT_F: TKMPointF = (X: 0.0; Y: 0.0);
```

Comment:
See comment below.
On a sidenote: let's refactor these into KMPOINT_ZERO and KMPOINT_INVALID, so they share the same starting letters and be logically grouped.